effekt/shared/src/main/scala/effekt/machine/Transformer.scala

```diff
@@ -224,6 +224,18 @@ object Transformer {
           Invoke(Variable(transform(id), transform(tpe)), opTag, values)
         }
 
+      case lifted.App(lifted.Member(lifted.Unbox(lifted.ValueVar(id, lifted.ValueType.Boxed(tpe))), op, annotatedTpe), targs, args) =>
```

Comment:
Is there a more principled way of doing this? I understand (somewhat), that we need to special case `App(Member(...))`, but shouldn't we ANF unbox before this translation -- or during this translation? Similar to how we unbox values?

Reply:
Yes, eventually we will have to